### ISSUE TITLE
Add titles to embedded iframes (YouTube)

### ISSUE THREAD

**Issue by atsansone** [dart-lang/site-www#5301]
Add `title` attribute to all `iframe` tags throughout the site. These most commonly appear with YouTube video embeds. The iframes without this attribute violates WCAG [H64](https://www.w3.org/TR/WCAG20-TECHS/H64).

**Error:** Iframe element requires a non-empty title attribute that identifies the frame.
[htmlcs: WCAG2AA.Principle2.Guideline2_4.2_4_1.H64.1](https://squizlabs.github.io/HTML_CodeSniffer/Standards/WCAG2/#sc2_4_1)

**Example:**
```html
<iframe style="max-width: 100%; width: 100%; height: 230px;" src="https://youtube.com/embed/xWV71C2kp38" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe>
```

**Location:**
(selector "html > body > div:nth-child(6) > div > main > div > div:nth-child(13) > div:nth-child(1) > div > iframe")

**Comment by atsansone** [comment]
The files currently affected include:

```nocode
src/language/extension-methods.md:
  16  
  17: <iframe width="560" height="315"
  18  src="https://www.youtube.com/embed/D3j0OSfT9ZI"

src/null-safety/migration-guide.md:
  40  
  41: <iframe width="560" height="315" src="https://www.youtube.com/embed/eBr5tlumwlg" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
  42  

src/resources/videos.md:
  21  
  22: <iframe width="560" height="315" src="https://www.youtube.com/embed/videoseries?list={{page.dart-playlist-id}}" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
  23  

src/tools/dart-fix.md:
  20  
  21:   <iframe width="560" height="315" 
  22    src="https://www.youtube.com/embed/OBIuSrg_Quo" title="Using 'dart fix' YouTube video" 

src/tools/pub/publishing.md:
  18  
  19: <iframe width="560" height="315" src="https://www.youtube.com/embed/8V_TLiWszK0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
  20  
```
